cluster/gce/gci/configure-helper.sh

```diff
@@ -1269,9 +1269,12 @@ EOF
 function create-kubeconfig {
   local component=$1
   local token=$2
-  echo "Creating kubeconfig file for component ${component}"
-  mkdir -p "/etc/srv/kubernetes/${component}"
-  cat <<EOF >"/etc/srv/kubernetes/${component}/kubeconfig"
+  if [[ -e "${KUBE_HOME}/bin/gke-internal-configure-helper.sh" ]]; then
+    gke-internal-create-kubeconfig "${component}" "${token}"
```

Comment:
this is hard to follow and means a separate implementation has to match the `/etc/srv/kubernetes/${component}/kubeconfig` path constructed here

Reply:
How about also passing the path into the internal function? Would that be clearer?

Reply:
yes

Reply:
Splitting this change to a different PR for dberkov.